prettier/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:6.9.1-slim
```

Comment:
maybe also update the node version? I haven't tested it.

Reply:
No expertise or opinions here from me. Happy to defer to you to try that out. If the test suite stays green I'd be fine with it.

Reply:
updated it to latest LTS version